src/server/index.d.ts

```diff
@@ -33,14 +43,15 @@ export interface NextAuthOptions {
   /** @docs https://next-auth.js.org/configuration/options#jwt */
   jwt?: any
   /** @docs https://next-auth.js.org/configuration/options#pages */
-  pages?: {
-    signIn?: string
-    signOut?: string
-    /** Error code passed in query string as ?error= */
-    error?: string
-    verifyRequest?: string
-    /** If set, new users will be directed here on first sign in */
-    newUser?: string
+  pages?: NextAuthPages
+  /** @docs https://next-auth.js.org/configuration/options#translations  */
+  translations?: {
```

Comment:
think we should just call it `locales` all the way
```suggestion
  locales?: {
```